cdap-ui/server.js

```diff
@@ -157,7 +158,8 @@ getCDAPConfig()
     sockServer.on('connection', function(c) {
       log.debug('[SOCKET OPEN] Connection to client "' + c.id + '" opened');
       // @ts-ignore
-      var a = new Aggregator(c);
+      var a = new Aggregator(c, { ...cdapConfig, ...securityConfig });
+      wsConnections.push(c);
```

Comment:
we should remove this connection on close

Reply:
@elfenheart I am doing it on SIGTERM handler (L184). Unless I am missing something

Reply:
So we are keeping all connections in memory and only clear on SIGTERM? So if the server runs for a long time, will it OOM?

Reply:
SIGTERM is when the server is terminated, right? But when individual connection get closed, we need to remove them from the list.